.env.example

```
@@ -0,0 +1,8 @@
+POSTGRES_DB=<required-database-name>
+POSTGRES_USER=<required-database-user>
+POSTGRES_PASSWORD=<required-strong-database-password>
+DATABASE_URL=postgres://<required-database-user>:<required-strong-database-password>@localhost:5432/<required-database-name>?sslmode=disable
+JWT_SECRET=<required-long-random-jwt-secret>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the POSTGRES_DB key

(UnorderedKey)